python/servo/devenv_commands.py

```diff
@@ -284,38 +274,32 @@ def ndk_gdb(self, release, target):
         return p.wait()
 
     @Command('try',
-             description='Runs try jobs by force pushing to personal fork try branches',
+             description='Runs try jobs by force pushing to try branch',
              category='devenv')
     @CommandArgument(
-        'jobs', default=["try"], nargs='...',
-        help="Name(s) of job(s) (ex: try, linux, mac, windows, wpt)")
-    def try_jobs(self, jobs):
-        branches = []
-        # we validate branches because force pushing is destructive
-        for job in jobs:
-            # branches must start with try-
-            if "try" not in job:
-                job = "try-" + job
-            if job not in VALID_TRY_BRACHES:
-                print(job + " job doesn't exist")
-                return -1
-            branches.append(job)
-        remote = "origin"
-        if "servo/servo" in subprocess.check_output(["git", "config", "--get", "remote.origin.url"]).decode():
-            # if we have servo/servo for origin check try remote
-            try:
-                if "servo/servo" in subprocess.check_output(["git", "config", "--get", "remote.try.url"]).decode():
-                    # User has servo/servo for try remote
-                    print("You should not use servo/servo for try remote!")
-                    return -1
-                else:
-                    remote = "try"
-            except subprocess.CalledProcessError:
-                print("It looks like you are patching in upstream servo.")
-                print("Set try remote to your personal fork with `git remote add try https://github.com/user/servo`")
-                return -1
-        for b in branches:
-            res = call(["git", "push", remote, "--force", f"HEAD:{b}"], env=self.build_env())
+        '--remote', '-r', default="origin",
+        help='git remote to use for try pushes')
+    @CommandArgument(
+        'try_string', default=None, nargs='...',
+        help="Try string")
+    def try_jobs(self, remote="origin", try_string=None):
+        # if jobs is empty we just push last commit to try and it will do full run
+        if try_string:
+            try_string = " ".join(try_string)
+            sys.path.insert(0, os.path.join(os.path.dirname(__file__), "..", "..", "etc"))
+            from python.try_parser import Config
+            conf = Config(try_string)
+            res = call(["git", "commit", "--allow-empty", "-m", f"{try_string}\n\n{conf.toJSON()}"],
+                       env=self.build_env())
```

Comment:
The full build environment isn't necessary here:

```suggestion
            config = Config(try_string)
            result = call(["git", "commit", "--allow-empty", "-m", f"{try_string}\n\n{conf.toJSON()}"])
```

Reply:
Last time it didn't find git without this, but it work now (at least on my windows machine).